passphraseme

```diff
@@ -22,7 +22,18 @@ def generate(word_count=7):
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
     parser.add_argument('word_count', metavar='word_count', default=7, help='Word count')
+    parser.add_argument('-d', '--dictionary', nargs='?', metavar='dictionary', help='Dictionary file')
     args = parser.parse_args()
 
+    if args.dictionary is not None:
+        try:
+            with open(args.dictionary) as f:
+                words = [line.rstrip('\n') for line in open(args.dictionary)]
+        except FileNotFoundError:
+            print("The file doesn't exist.")
+            sys.exit()
+    if not args.word_count.isdigit():
+        print("Please input a valid number.")
+        sys.exit()
```

Comment:
This should probably be part of a separate commit. Also, a more Pythonic way of doing this would be to catch the ValueError that would happen at line 38, imho.